README.md

```diff
@@ -5,6 +5,68 @@ https://flutter.dev/
 Flutter in an open Source SDK for creating high-performance mobile apps for IOS and Android.
 The Flutter makes it easier for you to build user interfaces, while reducing the amount of code required to create and update your app.
 
+## Why use Flutter?
+
+- Flutter can be used to build applications for Android and IOS
+- The Dart language used in Flutter is quite similar to Java
+- We can update the application in real time
+- If we close the application when we open it again we can continue from where we were
+- Flutter benefits from several objects that help us to facilitate the construction process
+
+## Core Principles
+
+Flutter is designed for the creation of 2D mobile applications.</br> 
+It contains several widgets and tools that facilitate the realization of an app.</br>
+
+## Widgets
+
+Widget is a component that can be used to simplify the process of building applications.</br>
+It usually contains windows, buttons, icons, menus, scroll bars and other features.</br>
+
+A widget can define:
+- Physical elements of an application (buttons, menus or bars)
+- Visual elements such as colors  
+- Elements of the application aspect such as edges or the padding of the screen
+
+Widgets form a hierarchy whenever we use one of them we will have another that will complement it.</br>
+There are widgets that are used in all applications and some that are used in specific situations.</br>
+
+![Screen Shot 2020-02-06 at 19 01 17](https://user-images.githubusercontent.com/27420533/73969408-4475d280-4913-11ea-8384-99c863321155.png)
+
+They are composed of several other small widgets that have only one function but if we add them we will have a functional application.</br>
+For example the Container contains other widgets that have functions like layout, placement and size.</br>
+
+You can control the Widget layout using these other smaller widgets.</br>
+
+There are some widgets that have no physical form within the application instead their goal and control some aspects of another Widget.</br>
+Like: padding, alignment, row, columns, and grids.</br>
+
+## Layers
+
+The Flutter is formed by several layers that are each layer and built one on top of the other.
+
+
+![Screen Shot 2020-02-07 at 09 06 06](https://user-images.githubusercontent.com/27420533/74015797-36629900-4989-11ea-8ec1-757aecad18ce.png)
+
+The top layers are normally used more often than the bottom layers even if they have the same importance within the application.</br>
+
+The great goal of Flutter is to be organized in this way and reduce the code allowing us to use the different Widgets within the layers always forming a satisfactory result for the user.</br>
+
+## Building Widgets
+
+To be able to use the widgets we always have to use the build() function that will later allow us to use all the widgets needed to build the application.</br>
+
+For example, the menu bar should have the build() function to be able to carry all the buttons or some text.</br>
```

Comment:
```suggestion
For example, the `appBar` menu needs to be invoked using the **`build()`** function. 
It can then have other nested "child" widgets such as buttons or text.</br>
```